tests/4.5/target/test_target_map_global_arrays.c

```diff
@@ -1,33 +1,40 @@
-//===--- test_target_map_array.c - test mapping an array to the device ------===//
+//===------------------- test_target_map_global_arrays.c---------------------===//
```

Comment:
Usually we keep the file title "left-aligned" so it looks something like this:
`//===--- test_target_map_global_arrays.c ------------------------------------===//`
Three dashes and a space before the filename, and then a space and dashes after until it matches the length of the footer. Some tests omit the space after the filename... not sure how strictly we are enforcing this tho.